### DISCUSSION TITLE
"You might also like" section in general beatmap listing

### DISCUSSION THREAD

**Discussion by Acumena** [ppy/osu#19046] [Ideas]
It would recommend you beatmaps based on what type of music it thinks you're a fan of.

It would check your favorite beatmaps and all of the beatmaps you have given a rating then it would make a guess on what type of music/artists you like or don't like.

It would work in a similar way to how video streaming services recommend you videos based on your subscriptions and likes.

**Comment by nagi-desuuu** [comment]
So they would have to develop a brand new algorithm entirely for osu,
without realizing that the workload for the servers would essentially
double...

**Comment by Avenred** [comment]
A section for other beatmaps featuring the same artist, or maybe a section for other beatmaps from by same mapper would be nice. I don't think there really needs to be an entire "algorithm".
